### PR TITLE
Followup edits to PR 2875

### PR DESCRIPTION
**Pull request by bfallonf**
#2875

I don't think this topic has had some attention in some time, so I did a few more followup edits than in the original.

Either way, peer review plz?

@adellape @ahardin-rh @tpoitras

### CODE REVIEW

File: dev_guide/app_tutorials/maven_tutorial.adoc [hunk 1]
```diff
@@ -28,54 +28,57 @@ capability, read
...
 and the password is *admin123*.
 
+[NOTE]
+====
 Nexus comes pre-configured for the Central Repository, but you may need others
-for your application.  For many Red Hat images, it is recommended to add the
-*jboss-ga* repository at *_https://maven.repository.redhat.com/ga/_*.
-link:https://books.sonatype.com/nexus-book/reference/config-maven.html[Learn how to add repositories].
+for your application. For many Red Hat images, it is recommended to link:https://books.sonatype.com/nexus-book/reference/config-maven.html[add the
+*jboss-ga* repository] at *_https://maven.repository.redhat.com/ga/_*.
```
Comment (tpoitras) [inline dev_guide/app_tutorials/maven_tutorial.adoc]: why is this URL marked-up like a file path?

File: dev_guide/app_tutorials/maven_tutorial.adoc [hunk 2]
```diff
@@ -137,40 +140,41 @@ $ oc get pods -w
 [[nexus-connecting-to-nexus]]
 == Connecting to Nexus
 
-The next steps demonstrate defining a build that will use the new Nexus
-repository.  The rest of the tutorial will be using
-link:https://github.com/openshift/jee-ex.git[this example repository]
-with *wildfly-100-centos7* as a builder, but these changes should work for any
+The next steps demonstrate defining a build that uses the new Nexus repository.
+The rest of the tutorial will be using
```
Comment (tpoitras) [inline dev_guide/app_tutorials/maven_tutorial.adoc]: s/will be using/uses

**Comment by tpoitras** [comment]
@bfallonf 
couple small things, otherwise LGTM!
:incoming_envelope: :ox: :alarm_clock: :elephant: :fire_engine: :sunrise_over_mountains: :oil_drum:

**Comment by bfallonf** [comment]
@tpoitras Done. Thanks. Merging.

**Pull request merged**